SPECS/expat/expat.spec

```diff
@@ -1,14 +1,13 @@
 Summary:	An XML parser library
 Name:		expat
-Version:	2.2.6
-Release:    4%{?dist}
+Version:	2.4.1
+Release:    1%{?dist}
 License:	MIT
 URL:		https://libexpat.github.io/
 Group:		System Environment/GeneralLibraries
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        https://github.com/libexpat/libexpat/releases/download/R_2_2_6/%{name}-%{version}.tar.bz2
-Patch0:         CVE-2018-20843.patch
+Source0:        https://github.com/libexpat/libexpat/releases/download/R_2_4_1/%{name}-%{version}.tar.bz2
```

Comment:
Nit: Would be nice to replace `R_2_4_1` in the `Source0` tag with a macro, so we can avoid manual updates of there. I'd suggest defining a macro like `%define underscore_version $(echo %{version} | cut -d. -f1-3 --output-delimiter="_")`.

Reply:
Great suggestion thanks.